src/lib/index.d.ts

```diff
@@ -5,6 +5,7 @@ import GleeConnection from './connection.js'
 import Glee from './glee.js'
 
 type WebSocketServerType = 'native' | 'socket.io'
+type HttpServerType = 'native' | 'socket.io'
```

Comment:
Can we have `socket.io` HTTP servers? I think these options need to be specified in the documentation. 🤔

Reply:
I think we only have a native HTTP adapter, better to remove the adapter parameter from the HTTP config object.